src/components/Main/Greetings.js

```diff
@@ -0,0 +1,24 @@
+import { Outlet } from "react-router-dom"
```

Comment:
Para que te muestre el children de tu componente cuando lo invocas hay que pasarselo como prop y hacer uso de el, este children corresponde al texto  Ludwig   de la invocacion en App.js <Greetings lang="de">Ludwig</Greetings> 

```suggestion
const language = ['de', 'en', 'es', 'fr'];
let hello = ['Hallo', 'Hi', 'Arre', 'Bonjour'];

const Greetings = ({ lang, children }) => {
  let findLang = language.indexOf(lang);
  let word = hello[findLang];
  return (
    <>
      <div style={{ borderStyle: 'solid', margin: 5 }}>
        <p>
          {word} {children}
        </p>
      </div>
    </>
  );
};

export default Greetings;```